lib/g10_certification_test_kit.rb

```diff
@@ -34,8 +34,8 @@ def required_inputs(prior_outputs = [])
 
 module G10CertificationTestKit
```

Comment:
Probably kind of a pain, but should we change module names, file names, etc to be 100% consistent.

And I know we don't really have plans to publish this as a gem, but it seems like a good idea to have the `.rb` file match gem name, etc, right?

Reply:
You are right, I forgot to do that. I do plan to actually publish this as a gem.